rest_auth_toolkit/serializers.py

```diff
@@ -1,6 +1,8 @@
 from django.conf import settings
 from django.contrib.auth import authenticate, get_user_model
 from django.utils.translation import gettext as _
+from django.core import exceptions
+import django.contrib.auth.password_validation as validators
```

Comment:
Simpler and cleaner (because it avoids confusion with django form validators and restframework validators):

```python
from django.contrib.auth import password_validation
```